src/main/java/com/vinyl/repository/ProductRepository.java

```diff
@@ -4,6 +4,9 @@
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface ProductRepository extends CrudRepository<Product, Integer> {
+    List<Product> findAll();
```

Comment:
this method is already available I think in the CrudRepository, no need to re-declare it

Reply:
it is, but it returns an Iterable and I preffered a List